### PR TITLE
format code

### PR DESCRIPTION
**Pull request by IDragonfire**
formatted with eclipse formatter

### CODE REVIEW

File: conftest.py [hunk 1]
```diff
@@ -17,14 +17,14 @@
 
 from PyQt4 import QtGui, QtCore
 
-@pytest.fixture(scope="function")
+@pytest.fixture(scope = "function")
```
Comment (Sheeo) [inline conftest.py]: I very much do not like this formatting.

**Comment by Sheeo** [comment]
Overall this doesn't really contribute anything.

Format files as you work on them, preferably in seperate commits.

**Comment by IDragonfire** [comment]
I suggest to one time format the repo,
otherwise unified diff of PR's get very ugly,
e.g. https://github.com/FAForever/client/pull/188/files

**Comment by Sheeo** [comment]
That PR you linked is a perfect example of why you should split your commits, especially those that only touch whitespace.

**Comment by IDragonfire** [comment]
I split it before, but you said, clean the history:
https://github.com/FAForever/client/pull/133

**Comment by Sheeo** [comment]
By 'clean' I didn't mean to squash everything into a single commit :)

**Pull request closed without merging**